src/android/Zoom.java

```diff
@@ -1258,6 +1318,11 @@ public void onMeetingUserLeave(List<Long> list) {
     @Override
     public void onMeetingUserUpdated(long l) {}
 
+    @Override
+    public void onInMeetingUserAvatarPathUpdated(long l) {
```

Comment:
Added implementation for some new interfaces which were added in the new meeting SDK